exasol/ds/sandbox/runtime/ansible/roles/jupyter/files/notebook/transformers/text_generation.ipynb

```diff
@@ -33,19 +33,6 @@
     "display(get_access_store_ui('../'))"
```

Comment:
<p>The open_pyexasol_connection should already activate the language container.</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/exasol/data-science-sandbox/pull/112/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>

Reply:
<p>How come???</p>

Reply:
The helper notebook opening the JupySQL includes the container activation. The text generation is the only notebook in the transformers series where I don't use the JupySQL. Here it's a normal pyexasol connection.

Reply:
it was my assumption, from what we have discussed last year, that open_pyexasol_connection does the activation, but maybe that got lost

Reply:
No, open_pyexasol_connection_connection does just what the name says - opens the connection. I categorically object to loading it with other stuff.

Reply:
ok, I can live with that for the moment